packages/backend/src/backend_factory.ts

```diff
@@ -157,5 +159,6 @@ export const defineBackend = <T extends DefineBackendProps>(
     ...backend.resources,
     createStack: backend.createStack,
     addOutput: backend.addOutput,
+    stack: backend.mainStack,
```

Comment:
```suggestion
    stack: backend.stack,
```

if we're going with just `stack` let's keep naming consistent on backend factory as well.